pkgs/by-name/ra/rav1e/package.nix

```diff
@@ -45,6 +45,8 @@ rustPlatform.buildRustPackage rec {
   # and linking it with cctools ld64.
   postPatch = lib.optionalString (stdenv.isDarwin && stdenv.isx86_64) ''
     substituteInPlace build.rs --replace-fail '.arg("-x")' '.arg("-S")'
+    # Thin LTO doesn’t appear to work with Rust 1.79. rav1e fail to build when building fern.
+    substituteInPlace Cargo.toml --replace-fail 'lto = "thin"' 'lto = "fat"'
```

Comment:
this seems ok to me just by looking at the commit that added `thin` https://github.com/xiph/rav1e/commit/9f6645a4427a13478250753f8986d6753854cf67 and the fact this is darwin x64 only.

Reply:
I wish I knew why thin LTO doesn’t work with rav1e when built with Rust 1.79. The crate that’s failing doesn’t even use LTO.